packages/KSelect/KSelect.vue

```diff
@@ -254,6 +273,20 @@ export default {
     testMode: {
       type: Boolean,
       default: false
+    },
+    /**
+     * A flag for autosuggest mode
+     */
+    autosuggest: {
+      type: Boolean,
+      default: false
+    },
+    /**
+     * Loading state in autosuggest
+     */
+    loading: {
+      type: Boolean,
+      default: false
```

Comment:
Again, we'll need to document this new prop in `select.md`

Also, does this only get utilized when the `autosuggest` prop is set to `true`? If yes, that needs to be included in the documentation.

Reply:
Yes, `loading` is used when `autosuggest` is enabled. I'm adding it in the docs.